### ISSUE TITLE
Asses Current Repo Code

### ISSUE THREAD

**Issue by piense** [rLoopTeam/eng-software-pod-gui#1]
Looks like stuff in here hasn't been modified in 5 months. Doesn't look like a bad start though.

**Comment by gordon13** [comment]
Yep its very old. This was started and left alone for a while, while we figured out how to communicate between ground station and PI
